curriculum/challenges/english/03-front-end-development-libraries/sass/create-reusable-css-with-mixins.md

```diff
@@ -8,125 +8,90 @@ dashedName: create-reusable-css-with-mixins
 
-Newer CSS features take time before they are fully adopted and ready to use in all browsers. As features are added to browsers, CSS rules using them may need vendor prefixes. Consider `box-shadow`:
+Mixins are particularly useful for writing reusable styles that you can include in various parts of your CSS without repeating code. For example, consider a scenario where you want to style multiple paragraphs with the same font size and bottom margin. You can use a mixin to achieve this.
 
-```scss
-div {
-  -webkit-box-shadow: 0px 0px 4px #fff;
-  -moz-box-shadow: 0px 0px 4px #fff;
-  -ms-box-shadow: 0px 0px 4px #fff;
-  box-shadow: 0px 0px 4px #fff;
-}
-```
-
-It's a lot of typing to re-write this rule for all the elements that have a `box-shadow`, or to change each value to test different effects. Mixins are like functions for CSS. Here is how to write one:
+Here's an example mixin:
 
 ```scss
-@mixin box-shadow($x, $y, $blur, $c){ 
-  -webkit-box-shadow: $x $y $blur $c;
-  -moz-box-shadow: $x $y $blur $c;
-  -ms-box-shadow: $x $y $blur $c;
-  box-shadow: $x $y $blur $c;
+@mixin prose($font-size, $spacing) {
+  font-size: $font-size;
+  margin: 0;
+  margin-block-end: $spacing;
 }
 ```
 
-The definition starts with `@mixin` followed by a custom name. The parameters (the `$x`, `$y`, `$blur`, and `$c` in the example above) are optional. Now any time a `box-shadow` rule is needed, only a single line calling the mixin replaces having to type all the vendor prefixes. A mixin is called with the `@include` directive:
+You can include this mixin in any CSS rule like this:
 
 ```scss
-div {
-  @include box-shadow(0px, 0px, 4px, #fff);
+p {
+  @include prose(1.2rem, 1rem);
 }
 ```
 
 # --instructions--
 
-Write a mixin for `border-radius` and give it a `$radius` parameter. It should use all the vendor prefixes from the example. Then use the `border-radius` mixin to give the `#awesome` element a border radius of `15px`.
+Write a mixin called `shape` that takes three parameters: `$width`, `$height`, and `$color`. Use this mixin to set the width, height, and background color of an element. Then, use the `shape` mixin to style the `#square` element with a width of `50px`, height of `50px`, and background color of `red`.
 
 # --hints--
 
-Your code should declare a mixin named `border-radius` which has a parameter named `$radius`.
-
-```js
-assert(code.match(/@mixin\s+?border-radius\s*?\(\s*?\$radius\s*?\)\s*?{/gi));
-```
-
-Your code should include the `-webkit-border-radius` vendor prefix that uses the `$radius` parameter.
-
-```js
-assert(
-  __helpers.removeWhiteSpace(code).match(/-webkit-border-radius:\$radius;/gi)
-);
-```
-
-Your code should include the `-moz-border-radius` vendor prefix that uses the `$radius` parameter.
+Your code should declare a mixin named `shape` which has three parameters: `$width`, `$height`, and `$color`.
 
 ```js
 assert(
-  __helpers.removeWhiteSpace(code).match(/-moz-border-radius:\$radius;/gi)
+  code.match(
```

Comment:
as you are changing the tests anyway, please use specific asserts instead of the generic assert, in this case `assert.match()` would be spot on